BloomGameStudio/BloomDiscordBot
# dev

**whats changed**

- Readme update
- RPC fallback and retry

**How to test**
- Set env vars for RPCs
- Modify snapshot urls to testnet.hub.xxx
- Change vote count to 1 instead of 5 for check
- Modify the vote duration on discord phase to 2 minutes
- Run bot and publish a proposal
